jetty-util/src/main/java/org/eclipse/jetty/util/BufferUtil.java

```diff
@@ -387,6 +403,29 @@ public static void append(ByteBuffer to, byte[] b, int off, int len) throws Buff
             flipToFlush(to, pos);
         }
     }
+    
+    /* ------------------------------------------------------------ */
+    /** Append bytes to a buffer.
+     * @param to Buffer is flush mode
+     * @param b bytes to append
+     * @throws BufferOverflowException if unable to append buffer due to space limits
+     */
+    public static void append(ByteBuffer to, byte[] b) throws BufferOverflowException
+    {
+        append(to,b,0,b.length);
+    }
+    
+    /* ------------------------------------------------------------ */
+    /** Append bytes to a buffer.
```

Comment:
Seems like a copy/paste flub.
Perhaps it should be "Append String UTF-8 bytes to a buffer" ?